### PR TITLE
Add Smartsheet

### PR DESCRIPTION
**Pull request by ktdickinson**
Add support for Smartsheet formula language. (First time using GitHub - hope I'm doing the pull request correctly! Files are in the below repository. Thanks!)

https://github.com/ktdickinson/Smartsheet_byKevinDickinson

### CODE REVIEW

**Comment by pryrt** [comment]
Good try, but not quite.

You've done four PR, two of which you closed, and none of which contained all the required files/information.

The steps that need to happen:
1. Create your own fork of the repo (you have done this)
2. Add the UDL's XML to the right directory _in your fork_
3. Update the JSON and MD files _in your fork_
4. Create a PR from your fork
    - from your fork's [master branch](https://github.com/ktdickinson/userDefinedLanguages/tree/master), 
    - click the down arrow on **Contribute** 
    - select **Open Pull Request**

See also the notes that I might make in your existing PR (which I am closing, since they won't work as-is)

File: udl-list.md [hunk 1]
```diff
@@ -235,6 +235,7 @@
 | [Skyrim Papyrus language](./UDLs/Skyrim-Papyrus_byJohnKostrzewski.xml) | Skyrim Papyrus language | John Kostrzewski |
 | [Smali](./UDLs/Smali_byJasi2169.xml) | Smali | Jasi2169 |
 | [SmallWorld Magik](./UDLs/SmallWorldMagik_byChristianTaton.xml) | SmallWorld Magik | Christian Taton |
+| [Smartsheet](.UDLs/Smartsheet_byKevinDickinson.xml) | Smartsheet | [Kevin Dickinson](https://github.com/ktdickinson/Smartsheet_byKevinDickinson) |
```
Comment (pryrt) [inline udl-list.md:238]: The first should be `[Smartsheet](./UDLs/Smartsheet_byKevinDickinson.xml)` -- note the slash between the `.` and the `UDLs`

**Review by pryrt — COMMENTED** [review]
This PR will not be merged.  See the detailed comments on what's needed, then re-submit

**Comment by ktdickinson** [comment]
Very helpful, thanks for the detailed notes!

**Comment by pryrt** [comment]
I ended up incorporating my notes to you into the CONTRIBUTING.md, so that future contributors will have a more-clear set of instructions on how to make a PR, if (like you) they haven't done it before.

**Comment by ktdickinson** [comment]
I saw that and I appreciate it! Thanks again for all the help.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**